Frontend/android-native/mobility-common/src/main/java/in/juspay/mobility/common/MobilityCommonBridge.java

```diff
@@ -2639,9 +2767,12 @@ private void setMarkerActionImage(String actionImage, String primaryText, View c
         }
     }
 
-    private void setMarkerPointerImage(String pointerImage, Boolean isInvisiblePointer, MarkerType markerType, View customMarkerView) {
+    private void setMarkerPointerImage(String pointerImage, Boolean isInvisiblePointer, MarkerType markerType, View customMarkerView, float markerRotation) {
         Context context = bridgeComponents.getContext();
         ImageView pointer = customMarkerView.findViewById(R.id.pointer_img);
+        if(markerRotation > 1.0) {
```

Comment:
what if we want to set the rotation to 0.0 intentionally?

Reply:
we can remove that